src/main/java/ua/ithillet/lesson6/WorkWithListCollection.java

```diff
@@ -0,0 +1,69 @@
+package ua.ithillet.lesson6;
+
+import java.util.*;
+
+public class WorkWithListCollection {
+
+    public static int countOccurrence(List<String> list, String searchItem) {
+        int count = 0;
+        for (String item : list) {
+            if (item.equals(searchItem)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List<Integer> toList(ArrayList<Integer> list) {
+        List<Integer> targetList = new ArrayList<>(list);
+        return targetList;
+
+    }
+
+    public static List<Integer> findUnique(List<Integer> list) {
+        List<Integer> uniqueList = new ArrayList<>();
+        try {
+            for (int i : list) {
+                for (int j : list) {
+                    if (i == j && !uniqueList.contains(i)) {
+                        uniqueList.add(i);
+                    }
+                }
+            }
+        } catch (RuntimeException ex) {
+            ex.printStackTrace();
+        }
+        return uniqueList;
+    }
+
+    public static List<String> calcOccurrence(List<String> list) {
+        List<String> countUniqueWords = new ArrayList<>();
+        List<String> result = new ArrayList<>();
+        for (String str : list) {
+            int count = 0;
```

Comment:
```suggestion
            int count = countOccurrence(list, str);
```
ця логіка вже реалізована в методі `countOccurrence `